its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -820,7 +820,11 @@ public void testIgnoreIssuesDoesNotRemoveSourceGenerator() throws IOException {
   public void whenEachProjectIsOnDifferentDrives_AnalysisFails() throws IOException {
     Path projectDir = TestUtils.projectDir(temp, "TwoDrivesTwoProjects");
 
-    int setupStatus = CommandExecutor.create().execute(Command.create("setup.bat").setDirectory(projectDir.toFile()), 60 * 1000);
+    int setupStatus = CommandExecutor.create().execute(
+      Command.create("setup.bat")
```

Comment:
I want to keep the scripts in those folders for maintainability - it's clear from the test folder structure that it needs a setup and a cleanup, and this is why the path is wrong inside the solution

Reply:
What about some `@before` and `@after` in this UT? Those projects can share the same drive scaffolded by the test

(I'm not sure what the exact @ annotations should look like, just guessing)